docker-compose.yaml

```diff
@@ -106,6 +107,8 @@ services:
     entrypoint: /bin/sh -c "/app/bin/wanda eval 'Wanda.Release.init()' && /app/bin/wanda start"
     volumes:
       - trento-checks:/usr/share/trento/checks
+    extra_hosts:
+      - host.docker.internal:host-gateway
```

Comment:
+1, as we are already using it elsewhere, but I'd recall this does not always work depending on your network config in docker (see https://github.com/docker/for-linux/issues/264#issuecomment-2053624963).
But... we are using it for dev, so it shouldn't be a big deal